bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Page.java

```diff
@@ -227,7 +227,7 @@ default String getTitle() {
     }
 
     /**
-     * If this page is associated with a Template, then this method will return the Template's client libraries categories.
+     * If this page is associated with a Template, then this method will return the Template's client libraries categories for the body tag.
      *
      * @return an array of client libraries categories; the array can be empty if the page doesn't have an associated template or if the
```

Comment:
Would communicate the fact that these are body client libraries in the return doc